matsim/src/test/java/org/matsim/utils/objectattributes/attributeconverters/CoordArrayConverterTest.java

```diff
@@ -0,0 +1,56 @@
+/* *********************************************************************** *
+ * project: org.matsim.*
+ * CoordArrayConverterTest.java
+ *                                                                         *
+ * *********************************************************************** *
+ *                                                                         *
+ * copyright       : (C) 2019 by the members listed in the COPYING,        *
+ *                   LICENSE and WARRANTY file.                            *
+ * email           : info at matsim dot org                                *
+ *                                                                         *
+ * *********************************************************************** *
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *   See also COPYING, LICENSE and WARRANTY file                           *
+ *                                                                         *
+ * *********************************************************************** */
+
+package org.matsim.utils.objectattributes.attributeconverters;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import org.matsim.api.core.v01.Coord;
+
+public class CoordArrayConverterTest {
+
+    @Test
+    public void testFromToString() {
+        final CoordArrayConverter converter = new CoordArrayConverter();
+        String a = "[(223380.4988791829;6758072.4280857295),(223404.67545027257;6758049.17275259),(223417.0127605943;6758038.021038004),(223450.67625251273;6757924.791645723),(223456.13332351885;6757906.359813054)]";
```

Comment:
Here numbers, have point decimal separators, as it should be.

Reply:
Agreed.